src/expr.rs

```diff
@@ -719,11 +722,7 @@ impl<'a> Rewrite for Loop<'a> {
             .rewrite(context, block_width, offset)
             .map(|result| {
                 format!("{}{}{}{}{}",
-                        label_string,
-                        self.keyword,
-                        pat_expr_string,
-                        block_sep,
-                        result)
+                        label_string, self.keyword, pat_expr_string, block_sep, result)
```

Comment:
Request for future PRs - please could you keep reformatting and new code in different commits please? It makes the new code easier to review.